packages/desktop-client/src/components/accounts/TransactionsTable.js

```diff
@@ -460,37 +454,64 @@ function PayeeCell({
   );
 }
 
-function CellWithScheduleIcon({ scheduleId, children }) {
+function PayeeIcons({
+  transaction,
+  transferAccount,
+  onNavigateToTransferAccount,
+  onNavigateToSchedule,
+  children,
+}) {
+  let scheduleId = transaction.schedule;
   let scheduleData = useCachedSchedules();
+  let schedule = scheduleData
+    ? scheduleData.schedules.find(s => s.id === scheduleId)
+    : null;
 
-  let schedule = scheduleData.schedules.find(s => s.id === scheduleId);
-
-  if (schedule == null) {
-    // This must be a deleted schedule
+  if (schedule == null && transferAccount == null) {
+    // Neither a valid scheduled transaction nor a transfer.
     return children;
   }
 
-  let recurring = schedule._date && !!schedule._date.frequency;
-
+  let colorStyle = { color: 'inherit' };
   let style = {
-    width: 13,
-    height: 13,
-    marginLeft: 5,
-    marginRight: 3,
-    color: 'inherit',
+    width: 23,
+    height: 23,
+    ...colorStyle,
   };
 
+  let onScheduleIconClick = () => onNavigateToSchedule(scheduleId);
+
+  let recurring = schedule && schedule._date && !!schedule._date.frequency;
+  let ScheduledIcon = () => (
+    <Button bare style={style} onClick={onScheduleIconClick}>
+      {recurring ? (
+        <ArrowsSynchronize style={colorStyle} />
```

Comment:
The icons will need to have a `width` property set on their `style` or else they will size themselves to be too big and the buttons will appear blank.

Reply:
Should I just pass the same style used in the button to the icons?

Reply:
You’d need to pass in a smaller width/height since the button has padding defined on it that reduces the available space for the icon. With the size you’ve got now, the button content area is 13×13px.

Reply:
Also fwiw the old arrow buttons were 10px wide

Reply:
…and there wasn’t that 5px padding between the icon and the text (for transfers, schedules did have it. ugh not sure which one to pick here, the schedule icon does look bad™ without the spacing, transfer arrow looks fine but when you hover it is a bit odd)

Reply:
Adjusted icon sizes to match the existing ones:
![image](https://github.com/actualbudget/actual/assets/20313680/3b7c1593-30cd-47f4-8085-c60efeb30c55)

Hovered:
![image](https://github.com/actualbudget/actual/assets/20313680/6e193472-b25c-4cda-9df2-5c1492576afd)

Existing:
![image](https://github.com/actualbudget/actual/assets/20313680/9b3ca4f1-bfce-464f-9d46-8ac931144262)

I think having the same padding for the schedules and transfer resulting in the texts lining up looks better. But let me know if I should adjust it to retain the existing padding.

Reply:
Having the text line up definitely looks better — good point.